docs/client/full-api/api/templates.md

```diff
@@ -41,6 +41,31 @@ Example:
 Now you can invoke this helper with `{{dstache}}foo}}` in the template defined
 with `<{{! }}template name="myTemplate">`.
 
+Helpers can accept positional and keyword arguments:
+
+```javascript
+Template.myTemplate.helpers({
+  displayName: function (firstName, lastName, kw) {
+    var prefix = kw.hash.title ? kw.hash.title + " " : "";
+    return prefix + firstName + " " + lastName;
+  }
+});
+```
+
+Then you can call this helper from template like this:
+
+```
+{{dstache}}displayName "John" "Doe" title="President"}}
+```
+
+You can learn more about arguments to helpers in [Spacebars
+Readme](https://atmospherejs.com/meteor/spacebars).
+
+Under the hood, each helper starts a new
```

Comment:
Under the hood, each helper starts a new [`Tracker.autorun`](#/full/tracker_autorun). When its reactive dependencies change, the helper is rerun.